components/asana/actions/add-task-to-section/add-task-to-section.mjs

```diff
@@ -1,40 +1,89 @@
 // legacy_hash_id: a_njiaY8
+import asana from "../../asana.app.mjs";
 import { axios } from "@pipedream/platform";
 
 export default {
-  key: "asana-add-task-to-section",
   name: "Add Task To Section",
   description: "Add task to section",
```

Comment:
```suggestion
  description: "Add a task to a specific, existing section. This will remove the task from other sections of the project. [See the docs here](https://developers.asana.com/docs/add-task-to-section)",
```